### PR TITLE
docs(connector kit): Restructure the adoption view

### PR DESCRIPTION
**Pull request by stephanbcbauer**
## Description

Since we currently don't follow the structure for the adoption view, this PR changes it to the expected one.

**new structure**

image

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [ ] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [ ] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: docs-kits/kits/tractusx-edc/docs/kit/adoption-view/page_adoption-view.md [hunk 1]
```diff
@@ -40,13 +64,106 @@ The main difference between the EDC and the previous connectors of the [IDSA][id
...
+
+#### Contract Definition
+
+By combining [`Assets`](#data asset) and Policies, Contracts for data offerings are defined. These Contracts need to be accepted by consuming participants (Connectors) for the data exchange to take place.
+
+#### Contract offer
+
+The contract offer is a representation of the [`ContractDefinition`](#contract-definition) for a specific consumer and serves as protocol for a data transfer object (DTO) for a particular contract negotiation. If a data consumer wants to conclude a binding data exchange contract based on the terms of a Data Offer, the data consumer can communicate such desire to the data provider by referencing to a specific Data Offer. Under German law, this constitutes a binding offer by the data consumer. For now, the dtaa consumer only has the option to accept all terms of a Data Offer (or not). The Data Exchange Process does not yet provide for the data consumer to make an offer that deviates from the terms of a Data Offer as set by the data provider.
```
Comment (jkbquabeck) [inline docs-kits/kits/tractusx-edc/docs/kit/adoption-view/page_adoption-view.md:123]:
```suggestion
The contract offer is a representation of the [`ContractDefinition`](#contract-definition) for a specific consumer and serves as protocol for a data transfer object (DTO) for a particular contract negotiation. If a data consumer wants to conclude a binding data exchange contract based on the terms of a Data Offer, the data consumer can communicate such desire to the data provider by referencing to a specific Data Offer. This constitutes a binding offer by the data consumer. For now, the dtaa consumer only has the option to accept all terms of a Data Offer (or not). The Data Exchange Process does not yet provide for the data consumer to make an offer that deviates from the terms of a Data Offer as set by the data provider.
```

File: docs-kits/kits/tractusx-edc/docs/kit/adoption-view/page_adoption-view.md [hunk 2]
```diff
@@ -13,24 +13,48 @@ The ConnectorKit provides a connector framework, based on the [Eclipse Dataspace
...
-- Existing standards and protocols ([GAIA-X][gaiax-url] and [IDSA][idsa-url]) are used as far as possible.
+### Mission
+
+Collaboration needs trust. Within Catena-X trust is built on an open-source architecture and components designed to support full sovereignity.
+The Connector KIT provides the single source of information and technology to perform trustful data exchange within Catena-X. The Connector KIT bundles the interaction patterns, relevant standards, APIs, and reference implementations for developers.
+
+## Concept
+
+Trust, interoperability, and data sovereignty are the objectives and values for secure and sustainable peer-to-peer data exchange between organizations and companies. The claim is data sovereignty: Whoever provides data to the Data Space retains control and decides individually who is involved in the data exchange, how, when, where and under what conditions. A corresponding concept was developed in the context of [Gaia-X][gaiax-url] and the [International Data Space Association][idsa-url]. __The essential software component is the EDC Connector.__
```
Comment (jkbquabeck) [inline docs-kits/kits/tractusx-edc/docs/kit/adoption-view/page_adoption-view.md:29]:
```suggestion
Trust, interoperability, and data sovereignty are the objectives and values for secure and sustainable peer-to-peer data exchange between organizations and companies. The claim is data sovereignty: Whoever provides data to the Data Space retains control and decides individually who is involved in the data exchange, how, when, where and under what conditions. A corresponding concept was developed in the context of [Gaia-X][gaiax-url] and the [International Data Space Association][idsa-url]. __The essential software component is the EDC.__
```

File: docs-kits/kits/tractusx-edc/docs/kit/adoption-view/page_adoption-view.md [hunk 3]
```diff
@@ -13,24 +13,48 @@ The ConnectorKit provides a connector framework, based on the [Eclipse Dataspace
...
+
+With the [EDC Connector][edc-url], a new communication component was introduced, which implements the following architectural principles:
```
Comment (jkbquabeck) [inline docs-kits/kits/tractusx-edc/docs/kit/adoption-view/page_adoption-view.md:31]:
```suggestion
With the [EDC][edc-url], a new communication component was introduced, which implements the following architectural principles:
```

File: docs-kits/kits/tractusx-edc/docs/kit/adoption-view/page_adoption-view.md [hunk 4]
```diff
@@ -13,24 +13,48 @@ The ConnectorKit provides a connector framework, based on the [Eclipse Dataspace
...
+- Separation of metadata and data. Enables high throughput rates for the actual data transfer.
+- Consistent semantics. Is the basis for interoperability and digital value creation.
+- Automation. As far as possible, all processes, starting with determining the identity, through ensuring the contractually agreed regulations to data transmission, are automated.
+- Standardization. Existing standards and protocols ([GAIA-X][gaiax-url] and [IDSA][idsa-url]) are used as far as possible.
+
+## Use Case
+
+The EDC Connector is the enabling component for all use cases within Catena-X and the only component to execute the exchange of operational data to Data Space participants. The base use case is therfore the exchange of data between to partners. Value creation along the automotive value chain is tied to processing data in specific contexts. Within Catena-X exist 10 use cases companies can participate in. Active participation in given use cases always requires the use of an [EDC Connector][edc-url]. Depending on the use case a participant chooses to participate in, further components will be required.
```
Comment (jkbquabeck) [inline docs-kits/kits/tractusx-edc/docs/kit/adoption-view/page_adoption-view.md:44]:
```suggestion
The EDC is the enabling component for all use cases within Catena-X and the only component to execute the exchange of operational data to Data Space participants. The base use case is therfore the exchange of data between to partners. Value creation along the automotive value chain is tied to processing data in specific contexts. Within Catena-X exist 10 use cases companies can participate in. Active participation in given use cases always requires the use of an [EDC][edc-url]. Depending on the use case a participant chooses to participate in, further components will be required.
```

File: docs-kits/kits/tractusx-edc/docs/kit/adoption-view/page_adoption-view.md [hunk 5]
```diff
@@ -40,13 +64,106 @@ The main difference between the EDC and the previous connectors of the [IDSA][id
...
 - The provision of possible offers (contract offering) which, on the one hand, stipulates the data offered and the associated terms of use (policies) in corresponding contracts.
 - An interface for manual selection of data and associated contract offers.
 - The actual data transfer via the data plane extension
 - The connection of software systems on the customer and provider side
 
+## Additional Resources
+
+### Catena-X Standards
+
+The Connector KIT builds on the [Catena-X Standards][Catena-X-Standards-url]. Every data consumer and provider in Catena-X is required to comply with these standards. The [EDC Connector][edc-url] builds on the following standards:
```
Comment (jkbquabeck) [inline docs-kits/kits/tractusx-edc/docs/kit/adoption-view/page_adoption-view.md:77]:
```suggestion
The Connector KIT builds on the [Catena-X Standards][Catena-X-Standards-url]. Every data consumer and provider in Catena-X is required to comply with these standards. The [EDC][edc-url] builds on the following standards:
```

File: docs-kits/kits/tractusx-edc/docs/kit/adoption-view/page_adoption-view.md (hunk 1 above)
Comment (jkbquabeck) [inline docs-kits/kits/tractusx-edc/docs/kit/adoption-view/page_adoption-view.md:123]:
```suggestion
The contract offer is a representation of the [`ContractDefinition`](#contract-definition) for a specific consumer and serves as protocol for a data transfer object (DTO) for a particular contract negotiation. If a data consumer wants to conclude a binding data exchange contract based on the terms of a Data Offer, the data consumer can communicate such desire to the data provider by referencing to a specific Data Offer. Under German law, this constitutes a binding offer by the data consumer. For now, the data consumer only has the option to accept all terms of a Data Offer (or not). The Data Exchange Process does not yet provide for the data consumer to make an offer that deviates from the terms of a Data Offer as set by the data provider.
```

**Review by jkbquabeck — CHANGES_REQUESTED** [review]
Looks good. Just some minor details to be changed.

**Comment by stephanbcbauer** [comment]
@jkbquabeck thx for the review. Please do it again.

**Review by jkbquabeck — APPROVED** [review]
Looks good to me.

**Pull request merged**